tests/templates/test_layers/test_basic_entangler.py

```diff
@@ -89,8 +79,7 @@ def circuit(weights):
             return [qml.expval(qml.PauliZ(i)) for i in range(n_wires)]
 
         expectations = circuit(weights)
-        for exp, target_exp in zip(expectations, target):
-            assert exp == target_exp
+        assert np.allclose(expectations, target, atol=tol, rtol=0)
```

Comment:
```suggestion
        np.testing.assert_allclose(expectations, target, atol=tol, rtol=0)
```

This gives nicer error messages when the test fails, making it easier to debug.